config/jobs/kubernetes/wg-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -112,11 +113,19 @@ periodics:
       - --updated=720h
       - --token=/etc/github-token/token
       - |-
-        --comment=Rotten issues close after 30d of inactivity.
-        Reopen the issue with `/reopen`.
-        Mark the issue as fresh with `/remove-lifecycle rotten`.
+        --comment=The Kubernetes project does not have enough contributors to adequately respond to all issues and PRs.
```

Comment:
This is sadly an accurate description =/
It'd be nice if there was a better way of saying it, but I can't think of one that is also clear and concise.

 @nikhita @cblecker @alisondy
y'all have any suggestions?

Reply:
Maybe we can reword it like `The present number of contributors for the Kubernetes project are not enough to adequately respond to all issues and PRs`. Saying the same thing passively makes it sound less sad to me : )

Reply:
I prefer active voice for clarity, but I liked the idea of saying "as of right now"

Because if there are enough people actively triaging, the bot won't comment on anything

changed wording in https://github.com/kubernetes/test-infra/pull/23094/commits/ff17dfd48e4053eb1509dfb794a708f32d40ed01